dts/arm/nuvoton/m46x.dtsi

```diff
@@ -404,6 +404,30 @@
 			#size-cells = <0>;
 			status = "disabled";
 		};
+
+		epwm0: epwm@40058000 {
+			compatible = "nuvoton,numaker-pwm";
+			reg = <0x40058000 0x37c>;
+			interrupts = <24 0>, <25 0>, <26 0>, <27 0>;
+			interrupt-names = "brake", "pair0", "pair1", "pair2";
```

Comment:
The "brake" interrupt is not used anywhere.  Can it be deleted?

Reply:
of cousre, I will remove this unused interrupt.